myadmin-service-impls/student-services/src/main/java/za/ac/unisa/myadmin/student/services/controllers/StudentRestServiceImpl.java

```diff
@@ -0,0 +1,71 @@
+package za.ac.unisa.myadmin.student.services.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.util.CollectionUtils;
+import org.springframework.util.StringUtils;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import za.ac.unisa.myadmin.common.exceptions.DoesNotExistException;
+import za.ac.unisa.myadmin.common.exceptions.InvalidParameterException;
+import za.ac.unisa.myadmin.common.exceptions.MissingParameterException;
+import za.ac.unisa.myadmin.common.exceptions.OperationFailedException;
+import za.ac.unisa.myadmin.student.services.dto.StudentInfo;
+import za.ac.unisa.myadmin.student.services.student.StudentService;
+
+import java.util.List;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+
+@RestController
+@RequestMapping({"/studentservices"})
+public class StudentRestServiceImpl {
+
+	@Autowired
+	@Qualifier("StudentService")
+	private StudentService studentService;
+
+	/**
+	 * Forgotten Student Number resource
+	 *
+	 * @param studentRequest
+	 * @return Valid student otherwise error response.
+	 * @throws OperationFailedException
+	 * @throws MissingParameterException
+	 * @throws InvalidParameterException
+	 * @throws DoesNotExistException
+	 */
+	@PostMapping(path = "/student/find", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
```

Comment:
Normally a query request is a GET operation. I would like us to conform to the standard of naming the search operation rest URI as "path="/students" e.g. [ExaminationRestServiceImpl](https://github.com/UnisaProject/UnisaMyAdmin/blob/master/myadmin-service-impls/exam-services/src/main/java/za/ac/unisa/myadmin/exam/services/controllers/ExaminationRestServiceImpl.java#L31)